doc/nrf/device_guides/working_with_nrf/nrf70/developing/raw_tx_operation.rst

```diff
@@ -0,0 +1,118 @@
+
+TX injection mode can be enabled when the nRF70 Series device operates in Station mode, both when it is operating as connected Station or as an unconnected Station (i.e. not connected to Access Point).
+
+If TX injection mode is enabled, a channel of operation also has to be configured. This channel will be used as the default transmit channel when the device is **not operating as a connected Station**. When the device is **operating as a connected Station**, the Station connected channel will be used for RAW 802.11 packet transmission.
+
+The net_mgmt API ``NET_REQUEST_WIFI_CHANNEL`` can be used to set the desired channel for raw 802.11 packet transmission.
+
+See the :ref:`wifi_shell_sample` sample for more information on configuring mode setting and channel setting for raw packet transmission via shell commands.
+
+The shell commands and network management APIs to swtich between operational modes and to set channel are provided below
+
+.. list-table:: Wi-Fi raw packet transmission network management APIs
+   :header-rows: 1
+
+   * - Network management APIs
+     - Wi-Fi shell command
+     - Example Usage
+     - Description
+   * - net_mgmt(NET_REQUEST_WIFI_MODE)
+     - wifi mode -i<interface instance> <configuration>
+     - wifi mode -i1 -st
+     - configure interface instance 1 to Station + TX injector mode
+   * - net_mgmt(NET_REQUEST_WIFI_MODE)
+     - wifi mode -i<interface instance> <configuration>
+     - wifi mode -i1 -s
+     - configure interface instance 1 to pure station mode
+   * - net_mgmt(NET_REQUEST_WIFI_CHANNEL)
+     - wifi channel -i<interface instance> <channel number to set>
+     - wifi channel -i1 -c6
+     - set raw transmission channel to 6 for interface 1 to be used in Station idle mode
+
+.. _ug_nrf70_developing_tx_injection_mode_usage_requirements:
+
+TX injection mode usage requirements
+************************************
+
+The raw packet to be transmitted by an application should be a fully conformant 802.11 packet as defined below.
+
+* The packet has a valid 802.11 MAC header
+* Any 802.11 Information Elements that are supposed to be part of the 802.11 frame
+* The FCS is not added to the packet by the application and is concatenated by the nRF70 Series device
+
+In addtion, it should provide transmit parameters that inform the nrf70 series Wi-Fi device driver on how the packet should be transmitted. It should also provide a packet marker identifying itself as a special packet which should be handled differently by the nrf70 series Wi-Fi driver. The transmit parameters and the packet marker together forms the raw packet transmit header. The raw packet transmit header is prepended to the 802.11 conformant raw packet before transmitting the packet to the lower layers by the application.
```

Comment:
```suggestion
In addition, the raw packet should provide transmit parameters that inform the nRF70 Series Wi-Fi device driver on how the packet should be transmitted. It should also provide a packet marker identifying itself as a special packet which should be handled differently by the nRF70 Series Wi-Fi driver. The transmit parameters and the packet marker together form the raw packet transmit header. The raw packet transmit header is prepended to the 802.11 conformant raw packet   before the packet is passed to the Wi-Fi driver by the application.
```

Reply:
just making sure @VivekUppunda : who is adding the prepended raw header? Is it the application, or is it done by the TX injection mode? Guess the first, but want to double check.

Reply:
The prepended raw header is to be created and attached by the application

Reply:
done